frontend/catalyst/third_party/oqc/src/CMakeLists.txt

```diff
@@ -39,7 +58,6 @@ set(OQC_LIBRARIES
 set_target_properties(rtd_oqc PROPERTIES BUILD_RPATH "$ORIGIN/../utils")
 target_link_directories(rtd_oqc PRIVATE ${runtime_lib})
 
-pybind11_add_module(oqc_python_module oqc_python_module.cpp)
 target_include_directories(oqc_python_module PRIVATE ${runtime_includes})
 
 add_dependencies(rtd_oqc oqc_python_module)
```

Comment:
What I don't see in this PR yet is a way to deal with line right below:
```
target_compile_definitions(rtd_oqc PUBLIC -DOQC_PY=\"$<TARGET_FILE_NAME:oqc_python_module>\")
```

Reply:
I am following the OpenQASM backend code as an example, there we do something similar:
https://github.com/PennyLaneAI/catalyst/blob/main/runtime/lib/backend/openqasm/CMakeLists.txt#L47
Does it mean that we have the same problem at then OpenQASM backend? Or is there something in the `CMakeLists.txt` file that I am missing? Any hints?

Reply:
Great question! Let me look into this, if the given path is relative and we place both modules in the appropriate location, it may work. Btw we need to update the make wheel command to also copy over the `oqc_python_module.so`.

Reply:
Just had a look at the generated build files, and it looks like 
```
target_compile_definitions(rtd_oqc PUBLIC -DOQC_PY=\"$<TARGET_FILE_NAME:oqc_python_module>\")
```
is translated into:
```
"/usr/bin/clang++ -DOPENQASM_PY=\\\"openqasm_python_module.so\\\"
```
So as long as the two libraries are shipped in the same directory we should be good 👍

Reply:
Fixed